static/app/views/stories/index.tsx

```diff
@@ -305,10 +169,21 @@ const StoryMainContainer = styled(VerticalScroll)`
   border-radius: ${p => p.theme.panelBorderRadius};
   border: 1px solid ${p => p.theme.border};
 
+  grid-area: body;
+
   padding: var(--stories-grid-space);
   padding-top: 0;
   overflow-x: hidden;
   overflow-y: auto;
 
   position: relative;
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    scroll-margin-top: ${space(3)};
```

Comment:
The margin is enforced so that we keep some margin from the top of the page and the title when users click on the link